portfolio/src/main/java/com/google/sps/servlets/FormServlet.java

```diff
@@ -75,10 +92,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     // Obtain a list of at most 'max' comments
-    Query query = new Query("Comment");
+    Query query = new Query("ImageComment");
     List<Entity> results = 
       datastore.prepare(query).asQueryResultList(FetchOptions.Builder.withLimit(max));
     ArrayList<Comment> comments = formCommentList(results);
+
+    System.out.println(comments);
```

Comment:
Some discussion on `System.out.println` vs a dedicated logger, `log4j` is very popular. I think I mentioned this in the pod sync today. Logs are actually quite useful on the server as it creates an audit trail and a set of clues to tracking down bugs in production. But a logger is greatly preferred vs just using println as you have fine grained control on what logs are visible.

https://stackoverflow.com/questions/2750346/logger-vs-system-out-println

Reply:
Sorry I forgot to remove the comment before committing--I will definitely check out loggers, the print statements have not been the easiest to work with.